roles/mrot_config/tasks/readme

```diff
@@ -0,0 +1,115 @@
+What is MROT??
```

Comment:
Please rename this file to , `README.md`

Reply:
Resolved. Name is update to README.md